cppcache/include/geode/Cache.hpp

```diff
@@ -254,7 +254,7 @@ class APACHE_GEODE_EXPORT Cache : public GeodeCache {
         bool readPdxSerialized,
         const std::shared_ptr<AuthInitialize>& authInitialize);
 
-  std::shared_ptr<CacheImpl> m_cacheImpl;
+  std::unique_ptr<CacheImpl> m_cacheImpl;
```

Comment:
This feels much better, avoiding any ambiguity about who's responsible for a given cache. Why did it take us so long.